src/dialects/mssql/query-generator.js

```diff
@@ -342,20 +340,27 @@ export class MsSqlQueryGenerator extends AbstractQueryGenerator {
   }
 
   removeColumnQuery(tableName, attributeName, options) {
-    if (options) {
+    options = options || {};
+
+    const dbVersion = this.sequelize.options.databaseVersion;
+    const isGreaterThanSQLServer2016 = semver.valid(dbVersion) && semver.gte(dbVersion, '13.0.0');
+    if (options && !isGreaterThanSQLServer2016) {
       rejectInvalidOptions(
-        'removeColumnQuery',
+        'addColumnQuery',
```

Comment:
This method is `removeColumnQuery`, not `addColumnQuery`

Reply:
lol sorry my bad